blog/2019-10-17-debezium-newsletter-02-2019.adoc

```diff
@@ -0,0 +1,68 @@
+= Debezium's Newsletter 02/2019
+ccranfor
+:awestruct-tags: [ community, news, newsletter ]
+:awestruct-layout: blog-post
+
+Welcome to the Debezium community newsletter in which we share blog posts, group discussions, as well as StackOverflow
+questions that are relevant to our user community.
+
+== Upcoming Events
+
+* https://jokerconf.com[JokerConf - Practical change data streaming uses cases with Apache Kafka and Debezium]
+* https://qconsf.com[QCon San Fransisco - Practical change data streaming use cases with Apache Kafka and Debezium]
```

Comment:
Here's the deep links: 

* https://jokerconf.com/en/2019/talks/6nkqqv8zj4gmujxmo64ov3/
* https://qconsf.com/sf2019/presentation/practical-change-data-streaming-use-cases-apache-kafka-debezium